api/v1alpha6/conversion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Kubernetes Authors.
+Copyright 2021 The Kubernetes Authors.
```

Comment:
Not sure where this came from. Maybe change it to 2023 instead?

Reply:
That will be after I deleted it and then copied it from elsewhere after realising that was a mistake 😂